service/src/test/java/uk/gov/hmcts/reform/fpl/utils/CaseDetailsHelperTest.java

```diff
@@ -41,4 +49,36 @@ void shouldThrowAnExceptionIfCaseNumberIsNull() {
 
         assertThat(exception.getMessage()).isEqualTo(EXCEPTION_MESSAGE);
     }
+
+    @Nested
+    class RemoveTemporaryFields {
+        private Map<String, Object> data = new HashMap<>();
+        private CaseDetails caseDetails = CaseDetails.builder().data(data).build();
+
+        @BeforeEach
+        void populateMap() {
+            data.put("key1", "some value 1");
+            data.put("key2", "some value 2");
+            data.put("key3", 3);
+        }
+
+        @AfterEach
```

Comment:
should not be needed, as each test is executed in new instance of test class

Reply:
Wasn't aware of that, removed.